### PR TITLE
New Crowdin translations

### PR DESCRIPTION
**Pull request by teolemon**

### CODE REVIEW

File: po/common/zh_CN.po [hunk 1]
```diff
@@ -775,7 +775,7 @@ msgstr "https://en.wiki.openfoodfacts.org"
 # Do not translate Open Beauty Facts but do translate Cosmetics
 msgctxt "footer_obf"
 msgid "Open Beauty Facts - Cosmetics"
-msgstr ""
+msgstr "公开美丽真相-化妆品"
```
Comment (stephanegigandet) [inline po/common/zh_CN.po]: OBF should not be translated

File: po/common/tr.po [hunk 2]
```diff
@@ -3707,7 +3707,7 @@ msgstr "Dosya alındı"
 
 msgctxt "nutriscore_calculation_details"
 msgid "Details of the calculation of the Nutri-Score"
-msgstr ""
+msgstr "Nutri-Skor hesaplamasının detayları"
```
Comment (stephanegigandet) [inline po/common/tr.po]: don't translate Nutri-Score

Comment (teolemon) [reply po/common/tr.po]: fixed. should propagate soon

File: po/common/zh_CN.po (hunk 1 above)
Comment (teolemon) [reply po/common/zh_CN.po]: fixed. should propagate soon

**Pull request merged**